features/old/accounts/invoices.feature

```diff
@@ -36,3 +36,41 @@ Feature: Show invoices from account's page (#16015909)
     And I follow "2011-01-00000001"
     # TODO: Properly check if order is opened
     Then I should see "Add"
+
```

Comment:
We should avoid at all cost adding new features under `features/old` and instead create a new feature located at a proper location and with proper steps.

Reply:
What would be the proper location and proper steps?

Reply:
then do we need to fix cucumber under this features/old?

Reply:
Actually, here we 're not adding new features, we're just adding a couple of scenarios to the existing feature file.
But I can move it to another place that might be suitable.